key-service.js

```diff
@@ -23,11 +24,13 @@ function factory(
     options = options || {};
     var identityId;
     var url;
+    var params = $httpParamSerializer(options.params)
     if(options.identity) {
-      url = basePath + '?owner=' + encodeURIComponent(options.identity.id);
+      url = basePath +
+        '?owner=' + encodeURIComponent(options.identity.id) + '&' + params;
```

Comment:
Logic starts to get tricky but might want to add owner param to the mix here and just do one serialize call. Would need to refactor to not do that below though.